DEVELOPER_GUIDE.md

```diff
@@ -92,7 +92,13 @@ sourceCompatibility = JavaVersion.VERSION_11
 targetCompatibility = JavaVersion.VERSION_11
 ```
 
-Download Java 11 from [here](https://adoptium.net/releases.html?variant=openjdk11).
+Download Java 11 from [here](https://adoptium.net/releases.html?variant=openjdk11). 
+
+Since Gradle has deprecated the usage of the JDKs below JDK-16, we recommend to use [JDK 17](#jdk-17) for development. If you still want to stick to JDK-11, please add `-Dorg.gradle.warning.mode=none` when invoking any Gradle build task from command line, for example:
+
+```
+./gradlew check -Dorg.gradle.warning.mode=none
+```
 
 #### JDK 14
```

Comment:
Is this section still accurate? I haven't had a JDK-14 installed in a long long time and I don't think we should encourage folks to go out and download a version that has been EOL for so long.

Reply:
Dropped it altogether, thanks @andrross !